client/app/stories/index.jsx

```diff
@@ -16,6 +22,29 @@ storiesOf('Logo', module)
     <Logo />
   ))
 
+storiesOf('ButtonFilled', module)
+.add('View', () => (
+    <ButtonFilled></ButtonFilled>
+  ));
+storiesOf('ButtonFilledM', module)
```

Comment:
Just like with Logo above, the Buttons should be a single story with multiple views; keeps it clean.